unit_tests/test_base_version.py

```diff
@@ -34,7 +36,8 @@ def test_master(self):
         scylla_repo = self.url_base + '/master/rpm/centos/2021-08-29T00:58:58Z/scylla.repo'
         linux_distro = 'centos'
         version_list = general_test(scylla_repo, linux_distro)
-        self.assertEqual(version_list, ['6.0'])
+        assert len(version_list) == 1
+        assert ComparableScyllaVersion(version_list[0]) > '5.4'
```

Comment:
Need to limit it to be below the possible enterprise versions:

```
'5.4' < ComparableScyllaVersion(version_list[0]) < '2018.1'
```

Reply:
Not that critical from POV

The whole idea is to break it away from changes in branches

Reply:
It is very simple and useful change where code example is already provided.
It covers possible bug where a scylla version may be returned from different scylla type.